app/trakt_tools.py

```diff
@@ -4,18 +4,23 @@
 import trakt
 import os
 
-def trakt_get_movies(config_path, plex, data, is_userlist=True):
+def trakt_get_movies(config_path, plex, data, list_type='userlist'):
```

Comment:
im changing tmdb in my next update to just take in method instead of is_userlist that way if theres more ways in the future it can be added easily

Reply:
we also keep plex tools clean that way

Reply:
this is all im doing in my update for tmdb and then handling the switch in the trakt method
```python
        elif "tmdb" in method:
            movies, missing = imdb_tools.tmdb_get_movies(config_path, plex, value, method)
```
i would just have 

```python
        elif "trakt" in method:
            movies, missing = trakt_tools.trakt_get_movies(config_path, plex, value, method)
```
and then handle to switch logic in the trakt method

Reply:
Sounds good, but I'm going to save the cleanup for another release. I want to be explicit about the method support, so that https://github.com/mza921/Plex-Auto-Collections/blob/22bd7da5b306cd5ba309b6a2e4237d7b10aea7db/app/plex_tools.py#L187 has a chance to execute.

Reply:
ok sounds good and dont forget to update the version under the title in the main file